docs/hsi.md

```diff
@@ -307,14 +337,16 @@ Converged Security and Manageability Engine is a standalone management module th
 The CSME lives in the PCH and can only be updated by the OEM vendor.
 The version of the CSME module can be checked to detect the most common and serious vulnerabilities: CVE-2017-5705, CVE-2017-5708, CVE-2017-5711, CVE-2017-5712, CVE-2017-5711, CVE-2017-5712, CVE-2017-5706, CVE-2017-5709, CVE-2017-5707 or CVE-2017-5710.
 
-Possible results:
+**Impact:** Using any one of the critical vulnerabilities, a remote hacker can take full control of the system and all connected devices, even when the system is powered off.
```

Comment:
I recommend using word "attacker" instead of "hacker".

Reply:
Fixed both places, thanks.